### PR TITLE
Align credentials stores fix

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

**Comment by maimorag** [comment]
not sure regarding the Jirav2, is the changes ok?

File: Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml [hunk 1]
```diff
@@ -76,6 +75,14 @@ configuration:
   type: 8
   section: Connect
   advanced: true
+- display: Incident type
```
Comment (JudahSchwartz) [inline Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml:78]: why move this?

**Review by JudahSchwartz — APPROVED** [review]

**Comment by JudahSchwartz** [comment]
Jirav2 is deprecated so i would leave it alone

File: Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml (hunk 1 above)
Comment (maimorag) [reply Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml:78]: validation error for the previous Incident type (IN124 I think)

**Comment by maimorag** [comment]
removed jira pack

**Pull request merged**